source/app/scorearea.h

```diff
@@ -56,11 +56,12 @@ class ScoreArea : public QGraphicsView
     std::shared_ptr<ClickPubSub> getClickPubSub() const;
 
     /// returns the palette used by scorearea
-    QPalette getPalette() const;
+    const QPalette *getPalette() const;
 
 protected:
     virtual void focusInEvent(QFocusEvent *event) override;
     virtual void focusOutEvent(QFocusEvent *event) override;
+    bool event(QEvent *event);
```

Comment:
Compiling on OSX, I get a warning here that the `override` keyword should be used.